qiskit/providers/aer/backends/qasm_simulator.py

```diff
@@ -253,14 +253,20 @@ class QasmSimulator(AerBackend):
       (Default: 1e-16).
 
     * ``mps_sample_measure_algorithm`` (str):
-      Choose which algorithm to use for ``"sample_measure"``. ``"mps_probabilities"``
-      means all state probabilities are computed and measurements are based on them.
-      It is more efficient for a large number of shots, small number of qubits and low
-      entanglement. ``"mps_apply_measure"`` creates a copy of the mps structure and
-      makes a measurement on it. It is more effients for a small number of shots, high
-      number of qubits, and low entanglement. If the user does not specify the algorithm,
-      a heuristic algorithm is used to select between the two algorithms.
-      (Default: "mps_heuristic").
+      Choose which algorithm to use for ``"sample_measure"``.
+      ``"mps_probabilities"``first constructs the probability vector and
+      then generates a sample per shot. It is more efficient for a large number
+      of shots and a small number of
+      qubits, with complexity O(2^n * n * D^2) to create the vector and
+      O(1) per shot, where n is the number of qubits and D
+      is the bond dimension.
+      ``"mps_apply_measure"`` creates a copy of the mps structure and
+      measures directly on it. It is more efficient for a small number of
+      shots, and a high number of qubits, with complexity around
+      O(n * D^2)
+      per shot.
+      (Default: "mps_apply_measure").
+
```

Comment:
If you make changes, following the previous comments, then don't forget to change here too. Is there a way to avoid this duplication in documentation?